openapi.yaml

```diff
@@ -255,8 +255,8 @@ components:
 
     UID:
       oneOf:
-      - type: integer
-      - type: string
+        - type: integer
+        - type: string
```

Comment:
The changed indentation here isn't YAML-compliant (see [yaml.org](https://yaml.org)).  This file should be restored to its previous version.